src/lambdas/confirmDownload/auditTemporaryS3LinkCreated.ts

```diff
@@ -1,9 +1,10 @@
 import { sendSqsMessage } from '../../sharedServices/queue/sendSqsMessage'
 import { currentDateEpochSeconds } from '../../utils/currentDateEpoch'
 import { getEnv } from '../../utils/getEnv'
-
+import { logger } from '../../sharedServices/logger'
 export const auditTemporaryS3LinkCreated = async (zendeskId: string) => {
   try {
+    logger.appendKeys({ zendeskId: zendeskId })
```

Comment:
You don't need this line, because the zendeskId is already appended to the context in the handler that calls this.

Reply:
removed